package.json

```diff
@@ -8,34 +8,32 @@
     "tinycolor2": "^1.4.1"
   },
   "devDependencies": {
```

Comment:
If you want to debate about it, you can open an issue.
Right now this is the default behaviour of `yarn add`, and it seems fine to me.